client/attest.go

```diff
@@ -24,6 +27,39 @@ type AttestOpts struct {
 	// firmware event log, where PCRs 0-9 and 14 are often measured. If the two
 	// logs overlap, server-side verification using this library may fail.
 	CanonicalEventLog []byte
+	// Indicates whether the AK certificate chain should be retrieved for validation.
```

Comment:
We should make it clearer here that this will potentially make external network calls. Maybe calling it `FetchCertChain`, and making it clear _which_ certs are being fetched (i.e. from `IssuingCertificateURL`) and how.